.github/workflows/build-triton-wheel.yml

```
@@ -118,6 +118,7 @@ jobs:
           docker exec -t "${container_name}" "${PYTHON_EXECUTABLE}"  -m pip install -U setuptools==67.4.0
           # Triton xpu build use GCC11
           if [[ "${BUILD_DEVICE}" == xpu ]]; then
+            docker exec -t "${container_name}" "${PYTHON_EXECUTABLE}"  -m pip install -U pybind>=2.13.1
```

@guangyey ?
```suggestion
            docker exec -t "${container_name}" "${PYTHON_EXECUTABLE}"  -m pip install -U pybind11>=2.13.1
```

https://github.com/intel/intel-xpu-backend-for-triton/blob/8d5374a3114fb922c4d8c7e2bb646ef24c1dee2b/python/pyproject.toml#L3C77-L3C93

---

Thanks.